### PR TITLE
Add project-request-limit repo to CI

### PR DESCRIPTION
**Pull request by tkashem**

### CODE REVIEW

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
+  container:
+    from: test-bin
+- as: verify
+  commands: make verify
+  container:
+    from: src
+- as: e2e-aws
```
Comment (tnozicka) [inline ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: missing verify-deps

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base
```
Comment (tnozicka) [inline ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: missing injection of builder image https://github.com/openshift/release/blob/6c6b25af3a4b20f6a45481b2a8b627773d347c9f/ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml#L17-L21

also this isn't rhel dockerfile

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
+  container:
+    from: test-bin
+- as: verify
+  commands: make verify
+  container:
+    from: src
```
Comment (tnozicka) [inline ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: from: bin

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
```
Comment (tnozicka) [inline ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: missing serial and upgrade test

**Comment by tnozicka** [comment]
I'd advise to start by taking already tuned config e.g. in https://github.com/openshift/release/blob/6c6b25af3a4b20f6a45481b2a8b627773d347c9f/ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml#L17-L21 and just dropping the operator e2e tests

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml (hunk 1 above)
Comment (tkashem) [reply ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: added

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml (hunk 2 above)
Comment (tkashem) [reply ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: @tnozicka added `injection of builder image`, although not sure why we need it. Is there any document that clarifies why we need it and what it does?

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml (hunk 3 above)
Comment (tkashem) [reply ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: fixed

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml (hunk 4 above)
Comment (tkashem) [reply ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: @tnozicka  I don't think the operand needs it unless it is a convention to add `serial and upgrade` across the board for all repos. 
May be the operator repo will have a use for these, although I doubt it. This operator is not a cvo enabled operator and is independent of OpenShift release. When the cluster upgrades, the operator (if installed) will be left untouched.

File: ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml (hunk 2 above)
Comment (sallyom) [reply ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml]: I think you can reference like so:
```
  bin:
    as:
    - builder
```

**Comment by petr-muller** [comment]
@tkashem please note that the rehearsals are not blocking the merge. You can totally ignore the build01 ones, too.

**Pull request merged**